### PR TITLE
fix: multitude of fixes

### PR DESCRIPTION
**Pull request by OlteanuCosmin**
~~C4R-343: Multiple API calls are made but not all are needed~~
C4R-708: Short description text of an app is not displayed entirely
C4R-428: Error is missing when invalid data are entered
C4R-717: The app card's buttons are not displayed properly
C4R-480: Some fields accept duplicate values
C4R-703: Persistent toast
C4R-699: The button position should be set at bottom of apps
C4R-401: Error occurs when user tries to delete his account
C4R-528: Phone number field accepts duplicate values at employee

### CODE REVIEW

File: backend/src/modules/organization/services/organization.service.ts [hunk 1]
```diff
@@ -703,6 +707,54 @@ export class OrganizationService {
       );
     }
 
+    const organizationWithEmail = await this.organizationGeneralService.findOne(
```
Comment (radulescuandrew) [inline backend/src/modules/organization/services/organization.service.ts:710]: Astea nu existau? 

@birloiflorian?

File: backend/src/modules/organization/services/organization.service.ts [hunk 2]
```diff
@@ -703,6 +707,54 @@ export class OrganizationService {
...
+
+    if (organizationWithEmail) {
+      errors.push(
+        new BadRequestException(
+          ORGANIZATION_REQUEST_ERRORS.CREATE.ORGANIZATION_EMAIL_EXISTS,
+        ),
+      );
+    }
+
+    const organizationWithAlias = await this.organizationGeneralService.findOne(
```
Comment (radulescuandrew) [inline backend/src/modules/organization/services/organization.service.ts:724]: Ar merge sa facem un singur query cu `email` or `phone` or `alias` si sa cautam in array? Vreun motiv pentru care n-am facut asa? (putem sa punem task for the moment)

File: backend/src/modules/user/services/user.service.ts [hunk 3]
```diff
@@ -113,29 +113,37 @@ export class UserService {
     try {
       const { applicationAccess, ...userData } = payload;
 
-      // 1. Check if the user exists
+      // 1. Check if user with received data exists
+      const userEmailCheck = await this.findOne({
```
Comment (radulescuandrew) [inline backend/src/modules/user/services/user.service.ts:117]: phoneCheck

File: backend/src/modules/user/services/user.service.ts [hunk 4]
```diff
@@ -149,21 +157,24 @@ export class UserService {
         case USER_ERRORS.GET.errorCode: {
```
Comment (radulescuandrew) [inline backend/src/modules/user/services/user.service.ts:157]:
```
case USER_ERRORS.GET.errorCode:
case USER_ERRORS.ACCESS.errorCode:
case USER_ERRORS.ALREADY_EXISTS_PHONE.errorCode:
throw new BadRequestException(err); 
``` 
Merge asa ca sa nu mai scriem 100 linii?

File: backend/src/modules/user/services/user.service.ts [hunk 5]
```diff
@@ -390,13 +405,26 @@ export class UserService {
         }
         // 2. USR_011: Error on assigning applications
         case USER_ERRORS.ACCESS.errorCode: {
-          throw error;
+          throw new BadRequestException({
```
Comment (radulescuandrew) [inline backend/src/modules/user/services/user.service.ts:408]: La fel ca celalalt comment

File: frontend/src/pages/create-organziation/configs/CreateOrganizationUserConfig.ts [hunk 6]
```diff
@@ -37,7 +37,7 @@ export const CreateOrganizationUserConfig: Record<string, any> = {
         message: translations.name.max,
       },
       minLength: {
-        value: 3,
+        value: 5,
```
Comment (radulescuandrew) [inline frontend/src/pages/create-organziation/configs/CreateOrganizationUserConfig.ts:40]: Why?

File: frontend/src/pages/users/components/UserCreate/UserCreate.tsx [hunk 7]
```diff
@@ -62,6 +63,22 @@ const UserCreate = () => {
           navigate('/users');
         },
         onError: () => {
+          const updateError: any = createUserMutation.error;
```
Comment (radulescuandrew) [inline frontend/src/pages/users/components/UserCreate/UserCreate.tsx:66]: @birloiflorian Ar trebui sa le mapam ca in CREATE_ORGANIZATION_ERRORS?

**Review by radulescuandrew — CHANGES_REQUESTED** [review]

File: frontend/src/pages/create-organziation/configs/CreateOrganizationUserConfig.ts (hunk 6 above)
Comment (OlteanuCosmin) [reply frontend/src/pages/create-organziation/configs/CreateOrganizationUserConfig.ts:40]: Nu corespundea cu valoarea din BE.

File: backend/src/modules/user/services/user.service.ts (hunk 4 above)
Comment (OlteanuCosmin) [reply backend/src/modules/user/services/user.service.ts]: Modificat.

File: backend/src/modules/user/services/user.service.ts (hunk 5 above)
Comment (OlteanuCosmin) [reply backend/src/modules/user/services/user.service.ts]: Modificat.

**Review by birloiflorian — APPROVED** [review]

**Pull request merged**